src/app/pages/onboarding/set-password/components/password-field.utils.ts

```diff
@@ -0,0 +1,39 @@
+import { color } from '@stacks/ui';
+
+import { PasswordStrengthScore, ValidatedPassword } from '@app/common/validation/validate-password';
+
+export const defaultColor = '#EFEFF2';
+
+enum PasswordStrengthText {
+  Poor = 'Poor',
+  Weak = 'Weak',
+  Average = 'Average',
+  Strong = 'Strong',
+}
```

Comment:
Here we could reuse the other enum. This is preferable because we avoid creating two structures of the same concept.

```suggestion
const passwordStrengthText = {
  [PasswordStrengthScore.Poor] = 'Poor',
  [PasswordStrengthScore.Weak] = 'Weak',
  [PasswordStrengthScore.Average] = 'Average',
  [PasswordStrengthScore.Strong] = 'Strong',
}
```

though, actually, given we're already using `PasswordStrengthScore` in the obj below, is this necessary at all? The text could go straight in the `strengthStyles` object

Reply:
Yep, that is true, I can move it there.